schema.sql

```diff
@@ -11,3 +11,29 @@ CREATE TABLE animals (
 
 alter table animals
 add column species varchar;
+
+create table owners (
+	id int generated always as identity,
+	full_name varchar,
```

Comment:
- [ OPTIONAL] 
It will be better if you make your data type VARCHAR to have a limit because it is a variable CHAR.
Example `full_name varchar(250),`